apps/pi-drone-detector/src/main.py

```
@@ -0,0 +1,184 @@
+        import json
+
+        data = json.dumps({
+            'event': 'drone_detected',
+            'detection': detection,
+        }).encode('utf-8')
+
+        req = urllib.request.Request(
+            webhook_url,
+            data=data,
+            headers={'Content-Type': 'application/json'},
+        )
```

Ruff (0.14.10)

Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)